service/s3/s3manager/upload.go

```diff
@@ -530,16 +533,17 @@ func (u *uploader) nextReader() (io.ReadSeeker, int, error) {
 		}
 
 		reader := io.NewSectionReader(r, u.readerPos, n)
+		r.Seek(int64(n), int(u.readerPos))
```

Comment:
Shouldn't the second arg be one of the `whence` constants?

Reply:
Oh wow, yeah you're totally right. The only reason this worked at all was because this method is only ever called when u.readerPos is 0. I'll change this to explicitly be 0.